Woosmap/expo_plugin_geofencing_batch
# Android implementation

## Issue
closes https://github.com/Woosmap/geofencing-react-native-plugin/issues/160

## Describe your changes
1. Added native implementation to push events to Batch
2. Modified sample app to integrate Geofencing Expo plugin and Batch Expo plugin

## How to test
1. 
```bash
npm install
cd example
npm install
cd ..
npm run build
```
2. Modify 

* `example/app.json` to add your Batch API Key
* `example/withBatchApiKey.js` to add your Batch API Key
* `example/App.tsx` -> `setWoosmapKey` method to add your Woosmap private key.

3.
```bash
cd example
npx expo run:android
```

4. Grant location permissions and start tracking. Check your Batch dashboard to confirm if the events are listed.

## Checklist:



## Comments
